### ISSUE TITLE
DEBUG: Something went wrong while running the command

### ISSUE THREAD

**Issue by wongzhiwei98** [ziirish/OctoPrint-TuyaSmartplug#46]
The plugin used to work for me until recently. Does anyone know what the issue is here regarding the command in the log?

Debug Logging Generated:
[2022-12-11 04:25:58,170] DEBUG: Checking status of Smart Switch.
[2022-12-11 04:25:58,172] DEBUG: Sending command: info to Smart Switch
[2022-12-11 04:25:59,334] DEBUG: Something went wrong while running the command
[2022-12-11 04:28:20,678] DEBUG: Checking status of Smart Switch.
[2022-12-11 04:28:20,679] DEBUG: Sending command: info to Smart Switch
[2022-12-11 04:28:20,854] DEBUG: Something went wrong while running the command
[2022-12-11 04:29:37,332] DEBUG: Checking status of Smart Switch.
[2022-12-11 04:29:37,333] DEBUG: Sending command: info to Smart Switch
[2022-12-11 04:29:38,573] DEBUG: Something went wrong while running the command
[2022-12-11 04:29:41,078] DEBUG: Checking status of Smart Switch.
[2022-12-11 04:29:41,078] DEBUG: Sending command: info to Smart Switch
[2022-12-11 04:29:41,235] DEBUG: Something went wrong while running the command
[2022-12-11 04:29:43,676] DEBUG: Checking status of Smart Switch.
[2022-12-11 04:29:43,676] DEBUG: Sending command: info to Smart Switch
[2022-12-11 04:29:44,930] DEBUG: Something went wrong while running the command

**Comment by swilsonnc** [comment]
Which hardware and os are you running octoprint on?

**Comment by wongzhiwei98** [comment]
[systeminfo.txt](https://github.com/ziirish/OctoPrint-TuyaSmartplug/files/10328637/systeminfo.txt)

Here is the system info generated. Let me know what info you need.

**Comment by swilsonnc** [comment]
The file you posted was blank.

**Comment by wongzhiwei98** [comment]
[systeminfo.txt](https://github.com/ziirish/OctoPrint-TuyaSmartplug/files/10338975/systeminfo.txt)

Sorry, please check the file attached here.

**Comment by wongzhiwei98** [comment]
Any update on this issue?

**Comment by swilsonnc** [comment]
Can you log into octopi cli and run python --version and paste the results?

**Comment by wongzhiwei98** [comment]
> Can you log into octopi cli and run python --version and paste the results?

![image](https://user-images.githubusercontent.com/72918152/213489020-a8d3aff3-f3a5-4ae3-817c-32bc48edcd30.png)

**Comment by swilsonnc** [comment]
What command are you using in octoprint (M81 ip_address)?  Is the ip address correct for the plug?  Is use version 3.3 checked in the plugin for the plug?

**Comment by wongzhiwei98** [comment]
> Inn your systeminfo.txt is says python version 3.7.3. I am guessing you are running octoprint in the venv and it is running python 3.7.3. Try running python3 --version. Either way if you are on python 3.7.3 then the plugin should work fine. Did anything change between it working and not working?

![image](https://user-images.githubusercontent.com/72918152/215044050-6a1bf17c-635e-4ed2-8a53-77dd1f41488b.png)

Is on Python 3.7.3. I do not think I change anything before it was working. Could it be anything to do with Tuya backend side regarding the keys and ID?

M81 command for my printer stated. It should be on its default setting, I didnt change anything for this command
![image](https://user-images.githubusercontent.com/72918152/215044582-764a28e3-86ba-4e43-92df-cb36ae5e8622.png)
My printer is still ON after running M81. My IP is correct cause I am using my Asus router to check the IP for each device. Version 3.3 is checked on the Smart Switch Tuya plugin configuration.

**Comment by wongzhiwei98** [comment]
Any update on this issue @swilsonnc ?

**Comment by swilsonnc** [comment]
Have you tried using the smart life app adding your plug and turning it off and on there.  That will at least let you know it works.Scott 
    	Get Outlook for iOS
    
  

 From: wongzhiwei98 ***@***.***>Sent: Monday, February 20, 2023 8:53 PMTo: ziirish/OctoPrint-TuyaSmartplug ***@***.***>Cc: swilsonnc ***@***.***>; Mention ***@***.***>Subject: Re: [ziirish/OctoPrint-TuyaSmartplug] DEBUG: Something went wrong while running the command (Issue #46) 
Any update on this issue @swilsonnc ?

**Comment by wongzhiwei98** [comment]
> 

Yes, I have tested with the smart life apps and the switch work normally without issue.

**Comment by swilsonnc** [comment]
Can you post a screenshot of your settings in octoprint?
    	Get Outlook for iOS
    
  

 From: wongzhiwei98 ***@***.***>Sent: Wednesday, February 22, 2023 8:21 PMTo: ziirish/OctoPrint-TuyaSmartplug ***@***.***>Cc: swilsonnc ***@***.***>; Mention ***@***.***>Subject: Re: [ziirish/OctoPrint-TuyaSmartplug] DEBUG: Something went wrong while running the command (Issue #46) 


Yes, I have tested with the smart life apps and the switch work normally without issue.

**Comment by wongzhiwei98** [comment]
> Can you post a screenshot of your settings in octoprint? Get Outlook for iOS  From: wongzhiwei98 ***@***.***>Sent: Wednesday, February 22, 2023 8:21 PMTo: ziirish/OctoPrint-TuyaSmartplug ***@***.***>Cc: swilsonnc ***@***.***>; Mention ***@***.***>Subject: Re: [ziirish/OctoPrint-TuyaSmartplug] DEBUG: Something went wrong while running the command (Issue #46)  Yes, I have tested with the smart life apps and the switch work normally without issue. —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

![image](https://github.com/ziirish/OctoPrint-TuyaSmartplug/assets/72918152/34e9cadd-d3ff-4732-9210-6a29d86c2c8b)

**Comment by wongzhiwei98** [comment]
I finally manage to get it working again already. I think the local key changed at Tuya side. I went a retrieve the key again and it work and solve the problem.
